doc/classes/Camera2D.xml

```diff
@@ -333,6 +333,7 @@
 			Bottom margin needed to drag the camera. A value of [code]1[/code] makes the camera move only when reaching the edge of the screen.
 		</member>
 		<member name="drag_margin_h_enabled" type="bool" setter="set_h_drag_enabled" getter="is_h_drag_enabled">
+			If [code]true[/code] this camera only moves when reaching the horizontal drag margins. If [code]false[/code] the camera moves horizontally regardless of margins. Default value: [code]true[/code].
```

Comment:
Everywhere else you say "the camera" instead of "this camera". You should make it consistent.